index.php

```diff
@@ -1,59 +1,59 @@
-        </div>
-    </div> <br>
-    <div class="server-content">
-        <div style="margin: 0; padding: 0;">
-            <h2 style="height: 10px">Paradise</h2>
-        </div>
-        <div style="font-size: 150%">
-            <p>New to Paradise Station or Space Station 13? <br>
-            <a href="http://nanotrasen.se/wiki/index.php/Guide_for_beginners">Read the Guide for Beginners!</a></p>
-        </div>
-        <div>
-            <a href="http://nanotrasen.se/phpBB3">Paradise Forums</a>
-        </div>
-        <div>
-            <a href="http://nanotrasen.se/wiki">Paradise Wiki</a>
-        </div>
-        <div class="social">
-            <div style="margin: 0; padding: 0;">
-                <h3 style="height: 10px">Paradise Social</h3>
-            </div>
-            <div class="discord">
-                <a href="https://discord.gg/0oOrkebBwhVkGX0f">Join us on Discord</a>
-            </div>
-        </div>
-    </div>
-
-    <footer>
-        <div style="font-size: 150%">
-            <p>These servers are hosted in Florida. <br>
-            Use the respective forums if you need to reach our staff.</p>
-        </div>
-    </footer>
-</div>
-
+    <div class="wrapper">
+    <nav>
+    
+       <ul>
+           <li id ="logo"><a  href="index.html"><img src="Images/Paradise2icon.PNG" alt="server logo"></a></li>
+           <li id="server"> <a href="byond://nanotrasen.se:6666">Server</a></li>
+           <li id="patreon"><a href="https://www.patreon.com/ParadiseStation">Patreon</a></li>
+           <li id="wiki">   <a href="https://nanotrasen.se/wiki/index.php/Main_Page">wiki</a></li>
+           <li id="forums"> <a href="https://nanotrasen.se/phpBB3/">forums</a></li>
+           <li id="discord"><a href="https://discord.gg/nuqD478">discord</a></li>
+           <li id="github"> <a href="https://github.com/ParadiseSS13/Paradise">github</a></li>
+       </ul>
+    </nav>
+    <main>
+        <h1>Paradise Station, a ss13 server</h1>
+        <div id="intro">
+            <p>
+                Paradise Station is a Space Station 13 server.
+                No idea what space station 13 is? Check out our new player guide <a href="https://nanotrasen.se/wiki/index.php/Guide_for_beginners"> here!</a>
+            </p>
+            <p>
+                We like to pride ourselves that the general playstyle on our server is a healthy mix or RP & action. So join us on the cyberiad, and try to stay alive, atleast for a while.
+            </p>
+       </div>
+        <ul>
+                <li>
+                   <h2>Forums & discord</h2>
+                   <p>Throughout the years, we've built a solid community around our server, with an active forum and a public discord server.</p> <p> The <a href="https://nanotrasen.se/phpBB3/">forums</a> are our official discussion space, where everthing can be document easily, while the <a href="https://discord.gg/nuqD478">discord</a> is a more informal space where players and admins can hang out.</p>
```

Comment:
@Draconic-saint It's HTML, and you're using <p> tags anyways, newlines won't have any effect on the final product (I think)